### PR TITLE
WIP Use /api/users endpoint instead of /api/identities for fetching list of users

### PR DESCRIPTION
**Pull request by alexeykazakov**
Fixes https://github.com/almighty/almighty-core/issues/672

### CODE REVIEW

File: src/app/user/user.service.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { User } from '../models/user';
 export class UserService {
   private headers = new Headers({'Content-Type': 'application/json'});
   private userUrl = process.env.API_URL + 'user';  // URL to web api
-  private identitiesUrl = process.env.API_URL + 'identities';  // URL to web api
+  private usersUrl = process.env.API_URL + 'users';  // URL to web api
```
Comment (alexeykazakov) [inline src/app/user/user.service.ts]: Not yet. And as I said in the description (see above ^) please do NOT merge until almighty/almighty-core#692 is merged ;)

Comment (sanbornsen) [inline src/app/user/user.service.ts]: Yeah, sorry I missed that. My bad.

File: src/app/user/user.service.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export class UserService {
     } else {
       logger.log('UserService running in production mode.');
     }
-    logger.log('UserService using user url ' + this.userUrl + ' identity url ' + this.identitiesUrl);
+    logger.log('UserService using user url ' + this.userUrl + ' users url ' + this.usersUrl);
```
Comment (debloper) [inline src/app/user/user.service.ts]: Do we need the logger at this point?

Comment (alexeykazakov) [inline src/app/user/user.service.ts]: I don't know but I didn't remove the original one. Just updated it accordingly.

**Comment by alexeykazakov** [comment]
https://github.com/almighty/almighty-core/pull/692 has been merged. This one should be merged now too.

**Comment by aslakknutsen** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

**Comment by sanbornsen** [comment]
Another PR is raised with the same changes and fixed tests - https://github.com/fabric8io/fabric8-planner/pull/930
Hence closing this PR.

**Pull request closed without merging**